### PR TITLE
:sparkles: adding lottery section

### PR DESCRIPTION
**Pull request by KipandaJr**
About the name of icons like `First, Second and Others`plz suggest other names or ignore this approach
@git-marcopitra @atilasousa @MarioBatalha 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206345322915680

### CODE REVIEW

**Review by git-marcopitra — CHANGES_REQUESTED** [review]
The name suggestion is object name or objective name

File: views/home/lottery/index.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import { FC } from 'react';
+
+import Button from '@/elements/button';
+
+import LeftIllustration from './illustraction/left-side';
+import RightIllustration from './illustraction/right-side';
+
+const LotterySection: FC = () => {
+  return (
+    <Box variant="container" mx="auto" id="lottery" bg="#1B1622" py="3rem">
```
Comment (MarioBatalha) [inline views/home/lottery/index.tsx:11]: You need to check the height in the way that the section fill the entire viewport aside of the coin address section.

File: views/home/lottery/index.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                mt="3rem"
+                fontWeight="700"
+                textAlign="center"
+                borderRadius="10rem"
+                border="3px solid #000"
+                boxShadow="10px 10px 0px 0px #000"
+                color="black"
+                left={['unset', 'unset', 'unset', 'unset', '-75%']}
+                fontSize={['1rem', '1.2rem', '1.5rem', '1.5rem', '2rem']}
+                p={[
```
Comment (MarioBatalha) [inline views/home/lottery/index.tsx:61]: Please check the padding, mainly px cuz it's not the same as on Figma

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: views/home/lottery/index.tsx (hunk 1 above)
Comment (KipandaJr) [reply views/home/lottery/index.tsx:11]: Burrrd for the time being you don't have to worry about being faithful to the LIB theme, you can skip these checks in BURRRD

File: views/home/lottery/index.tsx (hunk 2 above)
Comment (KipandaJr) [reply views/home/lottery/index.tsx:61]: Burrrd for the time being you don't have to worry about being faithful to the LIB theme, you can skip these checks in BURRRD

File: views/home/lottery/index.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
+import { Box, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import Button from '@/elements/button';
+
+import LeftIllustration from './illustraction/left-side';
+import RightIllustration from './illustraction/right-side';
+
+const LotterySection: FC = () => {
+  return (
```
Comment (git-marcopitra) [inline views/home/lottery/index.tsx:10]: remove the return

File: views/home/lottery/illustraction/coins/coin-wrapper.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+import { LotteryCoinWrapperProps } from '../../lottery.types';
+
+export const CoinWrapper: FC<LotteryCoinWrapperProps> = ({
+  Icon,
+  size,
+  ...props
+}) => {
+  const translateY = -7;
+  const delay = 0.5;
```
Comment (git-marcopitra) [inline views/home/lottery/illustraction/coins/coin-wrapper.tsx:13]: To have different timings on floating u can use `Math.random()` for the `delay` and `duration`

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Comment by git-marcopitra** [comment]
@KipandaJr write on the button "COMING SOON"

**Review by MarioBatalha — APPROVED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**